src/common/options.cc

```diff
@@ -4159,8 +4159,12 @@ std::vector<Option> get_global_options() {
 
     Option("osd_memory_target", Option::TYPE_SIZE, Option::LEVEL_BASIC)
     .set_default(4_G)
+    .set_min(128_M)
```

Comment:
I have now set the min to 896 MB based on the current values of osd_memory_base and osd_memory_cache_min. Also, I have added a long description mentioning how the min value is determined so that it acts like a guideline.